viz/index.js

```diff
@@ -145,73 +174,72 @@ const hideLoadingScreen = () => {
 // where presence of r in dicates showRace is UNchecked
 // where presence of i in dicates showIncome is UNchecked
 
-const URL_PARAM_CITY = 'city';
-const URL_PARAM_YEAR = 'year';
-const URL_PARAM_INCOME = 'i';
-const URL_PARAM_RACE = 'r';
-const URL_PARAM_BIKES = 'b';
+const URL_PARAM_CITY = "city";
+const URL_PARAM_YEAR = "year";
+const URL_PARAM_INCOME = "i";
+const URL_PARAM_RACE = "r";
+const URL_PARAM_BIKES = "b";
+const URL_PARAM_BORDER = "l";
```

Comment:
I suggest making this a more meaningfully named parameter.
e.g. "study_boundary"
const URL_PARAM_STUDY_BOUNDARY = "study-boundary"
and then add documentation for how to activate this feature in the README (see documentation for ?ha)